src/pages/PrintOnlyScreen.tsx

```diff
@@ -53,15 +55,17 @@ const PrintAppScreen = ({
     /* istanbul ignore else */
     if (isUsed) {
       await printer.print()
+      updateTally()
       printerTimer.current = window.setTimeout(() => {
         updateIsPrinted(true)
       }, printerMessageTimeoutSeconds * 1000)
     }
-  }, [markVoterCardPrinted, printer])
+  }, [markVoterCardPrinted, printer, updateTally])
 
   useEffect(() => {
     if (!isEmptyObject(cardVotes)) {
       window.localStorage.setItem(lastVotesKey, JSON.stringify(cardVotes))
+      // console.log(JSON.stringify(cardVotes))
```

Comment:
I know you'd want me to point out that you probably want to delete this commented out line :)